src/DssSpell.sol

```diff
@@ -48,36 +131,91 @@ contract DssSpellAction is DssAction {
+    address internal constant DAI_USDS      = 0x3225737a9Bbb6473CB4a45b7244ACa2BeFdB276A;
 
-    // ---------- Spark Proxy Spell ----------
-    // Spark Proxy: https://github.com/marsfoundation/sparklend-deployments/blob/bba4c57d54deb6a14490b897c12a949aa035a99b/script/output/1/primary-sce-latest.json#L2
-    address internal constant SPARK_PROXY = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;
-    address internal constant SPARK_SPELL = 0x668C84584Ef8EeEd6BFb4FFB2a4Fa03231F8b241;
+    address internal constant SUSDS         = 0xa3931d71877C0E7a3148CB7Eb4463524FEc27fbD;
+    address internal constant SUSDS_IMP     = 0x4e7991e5C547ce825BdEb665EE14a3274f9F61e0;
 
-    function actions() public override {
-        // ---------- Launch Project Funding ----------
-        // Forum: https://forum.makerdao.com/t/utilization-of-the-launch-project-under-the-accessibility-scope/21468/22
-        // MIP: https://mips.makerdao.com/mips/details/MIP108#9-1-launch-project-budget
-
-        // Launch Project - 9535993 DAI - 0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F
-        DssExecLib.sendPaymentFromSurplusBuffer(LAUNCH_PROJECT_FUNDING, 9_535_993);
+    address internal constant SKY           = 0x56072C95FAA701256059aa122697B133aDEd9279;
+    address internal constant MKR_SKY       = 0xBDcFCA946b6CDd965f99a839e4435Bcdc1bc470B;
 
-        // Launch Project - 2630.00 MKR - 0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F
-        MKR.transfer(LAUNCH_PROJECT_FUNDING, 2630.00 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+    address internal constant PAIR_DAI_MKR  = 0x517F9dD285e75b599234F7221227339478d0FcC8;
+    address internal constant PAIR_USDS_SKY = 0x2621CC0B3F3c079c1Db0E80794AA24976F0b9e3c;
 
-        // ---------- Spark Proxy Spell ----------
-        // Forum: https://forum.makerdao.com/t/aug-23-2024-proposal-changes-to-spark-for-upcoming-spell/24940
-        // Poll: https://vote.makerdao.com/polling/QmW55juU
-        // Poll: https://vote.makerdao.com/polling/QmQa73Cc
+    function actions() public override {
 
-        // Trigger Spark Proxy Spell at 0x668C84584Ef8EeEd6BFb4FFB2a4Fa03231F8b241
-        ProxyLike(SPARK_PROXY).exec(SPARK_SPELL, abi.encodeWithSignature("execute()"));
+        // Note: load the MCD contracts depencencies
+        DssInstance memory dss = MCD.loadFromChainlog(DssExecLib.LOG);
+
+        // ---------- New Tokens Init ----------
+        // Forum: TODO
+        // Poll: TODO
+        // MIP: TODO
+
+        // Init USDS by calling UsdsInit.init with the following parameters:
+        // Init USDS with usds parameter being 0xdC035D45d973E3EC169d2276DDab16f1e407384F
+        // Init USDS with usdsImp parameter being 0x1923DfeE706A8E78157416C29cBCCFDe7cdF4102
+        // Init USDS with UsdsJoin parameter being 0x3C0f895007CA717Aa01c8693e59DF1e8C3777FEB
+        // Init USDS with DaiUsds parameter being 0x3225737a9Bbb6473CB4a45b7244ACa2BeFdB276A
```

Comment:
As during LitePSM migration, I would suggest to add instructions directly on top of the actions + add new lines on top of them as per checklist. This will make it easier to read and verify that all instructions were taken and that no extra, unexplained actions are present

Reply:
That's what I'm doing actually. This whole block is a single instruction, with the difference that there are many parameters being specified, 1 by line.

Reply:
I meant to suggest a format with exactly 1 instruction and 1 action taken directly beneath it:
```solidity
        // Init USDS by calling UsdsInit.init with the following parameters:
        UsdsInit.init(

            // Note: DssInstance is required by the init library
            dss,

            UsdsInstance({
                // Init USDS with usds parameter being 0xdC035D45d973E3EC169d2276DDab16f1e407384F
                usds: USDS,

                // Init USDS with usdsImp parameter being 0x1923DfeE706A8E78157416C29cBCCFDe7cdF4102
                usdsImp: USDS_IMP,

                // Init USDS with UsdsJoin parameter being 0x3C0f895007CA717Aa01c8693e59DF1e8C3777FEB
                usdsJoin: USDS_JOIN,

                // Init USDS with DaiUsds parameter being 0x3225737a9Bbb6473CB4a45b7244ACa2BeFdB276A
                daiUsds: DAI_USDS
            })
        );
```
Like it was done for the LitePSM onboarding https://github.com/makerdao/spells-mainnet/blob/f750e59fc5b306bef3a8c1fbad3914854a8a0bb3/archive/2024-07-25-DssSpell/DssSpell.sol#L124-L176

This format allows [following checklist](https://github.com/makerdao/pe-checklists/blob/4b55876a9dec8b2c5ccaeb457fcbd134b629bb3a/spell/spell-reviewer-mainnet-checklist.md?plain=1#L22), and as explained above, it allows verifying that each instruction is taken and no extra, unexplained auction is present in the spell (like with `dss` parameter now)

Reply:
Alright. Trying to follow this approach in most of the content, however some instructions are declared in 1 line for better readability. I believe it would be better to not insert artificial line breaks for this purpose.